### ISSUE TITLE
Add MobileNetV3 performance to https://keras.io/api/applications/

### ISSUE THREAD

**Issue by sayakpaul** [keras-team/keras-io#907]
Not sure who contributed MobileNetV3 to `keras.applications`. Couldn't figure it out from https://github.com/keras-team/keras/commits/v2.9.0/keras/applications/mobilenet_v3.py either. 

The purpose of this thread is to just request the contributor of the model to add the following details and contribute a PR to https://keras.io/api/applications/:

* Size
* Top-1 Accuracy
* Top-5 Accuracy
* Parameters
* Depth (if applicable) 

The rest of the details (inference timings) can be still avoided since the hardware might vary (the official experiments were conducted on a specific machine configuration which may not be available to contributor). 

@LukeWood @fchollet @qlzh727

**Comment by LukeWood** [comment]
Perhaps a good course of action here is to wait until we produce tools that autogenerate these after training completes in the KerasCV example scripts.

**Comment by sachinprasadhs** [comment]
Does the below table satisfies your request?
![image](https://github.com/user-attachments/assets/59834fa4-5ba3-4c8e-a3f9-8059a8cfad9d)

https://keras.io/api/applications/mobilenet/
